docs/source/manual/client.rst

```diff
@@ -34,6 +34,10 @@ To create a :ref:`managed <man-core-managed>`, instrumented ``HttpClient`` insta
         public HttpClientConfiguration getHttpClientConfiguration() {
             return httpClient;
         }
+        
+        public void setHttpClientConfiguration(HttpClientConfiguration httpClient) {
```

Comment:
For the sake of completeness this should also be annotated with `@JsonProperty("httpClient")`.